website/styles/pages/admin.css

```
@@ -0,0 +1,302 @@
+    background: var(--bg-surface);
+    backdrop-filter: blur(12px);
+    border: 1px solid var(--glass-border);
+    border-radius: 12px;
+    padding: 1.5rem;
+    text-align: center;
+    transition: all 0.3s ease;
+}
+
+.stat-card:hover {
+    transform: translateY(-2px);
+    box-shadow: 0 8px 25px rgba(0, 0, 0, 0.1);
```

Stylelint (17.2.0)

[error] Expected "0.1" to be "10%" (alpha-value-notation)

(alpha-value-notation)